### ISSUE TITLE
Verbal Expression builder - IDE add-on or website

### ISSUE THREAD

**Issue by pashute** [VerbalExpressions/CSharpVerbalExpressions#35]
Make a Verbal Expression wizard, that helps you visually create and test the expressions validations and captures.

**Comment by jehna** [comment]
I suppose this is outside the scope of this project. You could start another project to create this functionality to IDEs or as a separate website tool

**Comment by MeaningOfLights** [comment]
The last remaining thing to make this a success is a manual, training guide, list of cool operations. Referring people to this project is hard as there is no documentation. An IDE would solve all this, happy to get involved!

**Comment by pashute** [comment]
No. It should be intuitive. Something like https://regex101.com/ 
or: http://regexr.com/

**Comment by Steven-Harris** [comment]
For anyone looking the javascript version has a site https://verbalregex.com/

**Comment by psoholt** [comment]
As noted, this is outside the project's scope. The JavaScript VerbalExpressions has a visual builder at verbalregex.com — a similar tool for the C# version would be welcome as a separate community project. Closing per the existing wontfix label.
